src/pages/installing/upgrading.md

```diff
@@ -1,26 +1,26 @@
 ---
 title: "Upgrading"
-excerpt: "Upgrading an ACD installation."
+excerpt: "Upgrading an IBM Watson ACD installation."
 categories: installing
 slug: upgrading
 toc: true
 ---
-The Annotator for Clinical Data Container Edition supports both manual and automatic upgrades of the operator and operand through [Operator Lifecycle Manager (OLM)](https://docs.openshift.com/container-platform/4.7/operators/understanding/olm/olm-understanding-olm.html#olm-overview_olm-understanding-olm) integration. OLM operators are limited to upgrades and do not rollback function. An operator upgrade is always done with a new replacement version. Operand upgrades are managed by the operator. An ACD instance is upgraded based on a specific set of related images that align with the ACD operator version.
 
-## Upgrade Paths
+_Note:  The upgrade path only applies to IBM Watson Annotator for Clinical Data Container Edition._
+
+The Annotator for Clinical Data Container Edition supports both manual and automatic upgrades of the operator and operand through [Operator Lifecycle Manager (OLM)](https://docs.openshift.com/container-platform/4.7/operators/understanding/olm/olm-understanding-olm.html#olm-overview_olm-understanding-olm) integration. OLM operators are limited to upgrades and do not rollback function. An operator upgrade is always done with a new replacement version. Operand upgrades are managed by the operator. An ACD instance is upgraded based on a specific set of related images that align with the ACD operator version.
 
-The update graph for ACD is expected to grow over time, but is very simple today. The major and minor version for ACD continues to be `2.0`. New function and fixes are released as patch versions made available as part of a continuous delivery strategy. The following patch versions are available today.
+## Upgrade paths
 
-- Upgrade the ACD operator from `v2.0.202106231922` to `v2.0.202109011707`.
-- Upgrade the ACD operator from `v2.0.202109011707` to `v2.0.202109100417`.
+The update graph for ACD is expected to grow over time, but is very simple today. The major and minor version for ACD continues to be `2.0`. New function and fixes are released as patch versions made available as part of a continuous delivery strategy. Refer here for a list of the [available versions](https://github.com/merative/acd-containers/blob/master/CHANGELOG.md#releases-from-ibm).
```

Comment:
There is no upgrade from IBM ACD to Merative ACD. See the migration overview for details. The update graph for Merative ACD will be provided when available. The major and minor version for ACD continues to be `2.0`. New function and fixes are released as patch versions made available as part of a continuous delivery strategy. Refer here for a list of the [prior IBM ACD versions](https://github.com/merative/acd-containers/blob/master/CHANGELOG.md#releases-from-ibm).